k8s/sidecar.yaml

```diff
@@ -41,14 +41,16 @@ spec:
         image: iptestground/sidecar:edge
         imagePullPolicy: Always
         command: ["testground"]
-        args: ["--vv", "sidecar", "--runner", "k8s", "--pprof"]
+        args: ["sidecar", "--runner", "k8s", "--pprof"]
```

Comment:
I guess it's fine reducing verbosity now that we have debugged and verified that it works well with sync v2.